chainguard-dev/edu
# IAM Updates - batch 1

## Type of change

This is the first batch of updates relating to recent IAM changes. Any references to IAM groups were changed to organizations and I renamed the `IAM & Groups` bucket to `IAM & Organizations`. I also added a directory redirect to reflect this change.

The most significant changes happened in the Overview doc where I added a section outlining what organizations are and deleted a reference to Gulfstream.

Otherwise I included changes that I caught in my editing process, mostly spelling errors.

### What should this PR do?

Does not resolve anything entirely

### Why are we making this change?

Reflecting changes to IAM structure + commands

### What are the acceptance criteria? 



### How should this PR be tested?

Please give all the changes a close look. 

Note that `npm run start` wasn't working for me (increasing verbosity and checking the logs weren't helpful?) so I fear there may be some outputs that have rendered poorly. I'll fix these in another commit if I can, assuming the deploy preview works correctly. 

EDIT: second commit should fix  these, feel free to flag any bad looking outputs or tables.
